subprojects/base-services/src/main/java/org/gradle/internal/os/OperatingSystem.java

```diff
@@ -362,7 +363,7 @@ protected String getArch() {
         }
 
         protected String getOsPrefix() {
-            String osPrefix = getName().toLowerCase();
+            String osPrefix = TextUtil.toLowerCaseUserLocale(getName());
```

Comment:
I'm not sure if it matters, but this should be english locale too to be consistent with the one above.